src/Illuminate/Contracts/Auth/Access/Gate.php

```diff
@@ -74,6 +74,24 @@ public function denies($ability, $arguments = []);
     public function check($ability, $arguments = []);
 
     /**
+     * Determine if any of the given ability should be granted.
+     *
+     * @param  iterable  $abilities
+     * @param  array|mixed  $arguments
```

Comment:
It is the way @taylorotwell has commented other methods as well. I chose to stick with his conventions.